setup.py

```diff
@@ -35,33 +35,25 @@
     keywords="framework knowledge graph data science",
     url="https://github.com/BlueBrain/nexus-forge",
     packages=find_packages(),
-    python_requires=">=3.8",
-    setup_requires=[
-        "setuptools_scm",
-    ],
+    python_requires=">=3.8.1",
+    setup_requires=["setuptools_scm"],
     install_requires=[
         "hjson",
         "pyyaml",
         "pandas",
         "nexus-sdk",
         "aiohttp",
-        "rdflib==6.3.2",
+        "rdflib==7.0.0",
```

Comment:
It seems that in spack we have up until 6.3.2:
https://github.com/BlueBrain/spack/blob/70cb1d988a88052205672d05984f87470afafd9c/var/spack/repos/builtin/packages/py-rdflib/package.py#L24

Reply:
Good catch. Not sure how easy is it to add 7.0.0 there. Is changing the following enough:

- `version("7.0.0", sha256="9995eb8569428059b8c1affd26b25eac510d64f5043d9ce8c84e0d0036e995ae")` from https://pypi.org/project/rdflib/#files
Also for pyshacl:
- version("0.25.0",sha256="91e87ed04ccb29aa47abfcf8a3e172d35a8831fce23a011cfbf35534ce4c940b")` in [py-pyshacl/package.py](https://github.com/BlueBrain/spack/blob/70cb1d988a88052205672d05984f87470afafd9c/var/spack/repos/builtin/packages/py-pyshacl/package.py) from https://pypi.org/project/pyshacl/#modal-close

And also changing the python requirement there.

If not that simple, then we can do the version bumps at next forge release as it is not required for having jsonld 1.1

Reply:
Given that these packages are builtins and not from BlueBrain a patch will need to be added. See:
https://github.com/BlueBrain/spack/blob/develop/bluebrain/documentation/packages.md#adding-new-versions-or-modifying-packages-with-non-bluebrain-code